src/dex/aave-v3/aave-v3-e2e.test.ts

```diff
@@ -1,23 +1,59 @@
 import dotenv from 'dotenv';
 dotenv.config();
 
+import { Interface } from '@ethersproject/abi';
 import { testE2E } from '../../../tests/utils-e2e';
-import {
-  Tokens,
-  Holders,
-  NativeTokenSymbols,
-} from '../../../tests/constants-e2e';
+import { Tokens, Holders } from '../../../tests/constants-e2e';
 import { Network, ContractMethod, SwapSide } from '../../constants';
 import { StaticJsonRpcProvider } from '@ethersproject/providers';
-import { getTokenFromASymbol } from './tokens';
-import { Token } from '../../types';
+import { getTokenFromASymbol, setTokensOnNetwork } from './tokens';
 import { generateConfig } from '../../config';
+import { fetchTokenList } from './utils';
+import { DummyDexHelper } from '../../dex-helper';
+import POOL_ABI from '../../abi/AaveV3_lending_pool.json';
+import ERC20ABI from '../../abi/erc20.json';
+import { Config } from './config';
 
 jest.setTimeout(1000 * 60 * 3);
 
 describe('AaveV3 E2E', () => {
   const dexKey = 'AaveV3';
 
+  beforeAll(async () => {
```

Comment:
Isn't this redundant with the [initializePricing](https://github.com/paraswap/paraswap-dex-lib/blob/master/tests/utils-e2e.ts#L229) call in testE2E ?

Reply:
Due to the way the tests are written this data is required to call getTokenFromASymbol(network, **pair.aTokenSymbol**) in order to pass parameters to testE2E(). If I removed these calls I would have to add hardcoded data in the tests.

The token list is cached, so it shouldn't affect test's performance.